packages/cli/src/utils/miscellaneous.ts

```diff
@@ -314,11 +314,17 @@ export function handleError(e: Error, ref: string) {
 
 export class HandledError extends Error {}
 
-export function printLintTotals(totals: Totals, definitionsCount: number) {
+export function printLintTotals(
+  totals: Totals,
+  definitionsCount: number,
+  documentSpecVersion?: string
+) {
   const ignored = totals.ignored
     ? yellow(`${totals.ignored} ${pluralize('problem is', totals.ignored)} explicitly ignored.\n\n`)
     : '';
 
+  const lintSubject = documentSpecVersion === 'arazzo' ? 'Test' : 'API';
```

Comment:
Arrazo is not a "Test". It can be used as a test.

Let's change the message to:

> Woohoo! Your description documents are valid.

@lornajane do you have any better suggestions?

Reply:
I think we could literally say the Arazzo description is valid if we're specifically testing for an arazzo file here. I think these are a little different to more interchangeable OpenAPI/AsyncAPI formats.

What do we output for config linting?

Reply:
It used to be this message `Woohoo! Your API description is valid. 🎉`.
But if we want to provide custom and different message for API and Arazzo - we will introduce a breaking change or additional redundancy to read document two times. The reason for this is to pass SpecVersion.
So we ended up wit more generic version for validation`Woohoo! Your description document is valid. 🎉`

@lornajane , please check previous discussion https://github.com/Redocly/redocly-cli/pull/1601#discussion_r1654752154

Reply:
I recommend we keep the existing `Woohoo! Your API description is valid. 🎉` feedback for existing formats (OpenAPI, AsyncAPI). I don't mind if we do the same or something different for Arazzo, but let's not disrupt the existing functionality.

Reply:
That's the problem. If we want to keep the existing message for existing format but have a different message for Arazzo we will have to introduce a breaking change in the API or re-design some internals.
I was hoping to avoid it so my suggestion is to have some generic message.

Reply:
I think we can keep the existing message for the sake of simplicity. Any Arazzo document could also be considered an API description to some extent. Later we can try customising the message if possible. 
WDYT?

Reply:
Yes! Good idea 😄

Reply:
Ok, I will roll back message and snapshots to prev state